ui/app/components/sidebar/frame.hbs

```diff
@@ -18,9 +18,11 @@
             />
           </:logo>
           <:actions>
-            <Hds::SideNav::Header::IconButton
+            <Hds::Button
+              @isIconOnly={{true}}
+              @size="large"
               @icon="terminal-screen"
-              @ariaLabel="Console toggle"
+              @text="Console toggle"
```

Comment:
I think it makes sense that the button has to have `@text` and that it's used for the aria, but maybe because of all the button work I've done 😂

Thanks for quoting the docs!